WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -27,7 +27,7 @@ public class WordPressComRestApi: NSObject
 
     public static let apiBaseURLString: String = "https://public-api.wordpress.com/rest/"
 
-    private let oAuthToken: String?
+    public let oAuthToken: String?
```

Comment:
It was probably a good idea to keep this `private`. Since the api is now Swift, we should make it conform to `Hashable` and use the api's `hashValue` instead of the authToken.